vmangos/core
# Add script for Archmage Tarsis Kir-Moldir 16381

CMaNGOS counterpart: https://github.com/cmangos/tbc-db/commit/cc4efc8f057d6fbdc472a1bf165bd456cb397552

## 🍰 Pullrequest
Archmage Tarsis is supposed to do a small script when the player first speaks with him. He closes the gossip menu, then kneels and sits before re-enabling gossip.
He also has a special interaction if the player has Frame of Atiesh (item 22727). 

### Proof
https://www.wowhead.com/classic/item=22727/frame-of-atiesh#comments:id=3297645
https://www.reddit.com/r/classicwow/comments/lc7eel/if_you_have_the_frame_of_atiesh_item_in_your_bags/
There was a video somewhere but I can't recall where. His regular gossip menu ids I used here are from classic sniffs.

### Issues
Wrong gossip menu ids were used in the existing data, so I cleaned those up as well as some locales_gossip_menu_option data which looks old/irrelevant now with broadcast text locales.

### How2Test
- Go to Naxx, talk to Archmage and witness his glorious script happen the first time you speak with him.
- Turn GM on or add Frame of Atiesh to see the alternate special gossip option and script.

### Todo / Checklist
- Add the special Frame of Atiesh interaction with Archmage Angela Dosantos 16116, maybe this one is missing as well.
